pages/basket/index.js

```diff
@@ -17,17 +17,19 @@ import { getProductBasedOnId } from "../../src/database/model";
 // }
 
 const basketArray = [
-  { product_id: 1, quantity: 2 },
-  { product_id: 3, quantity: 1 },
-  { product_id: 4, quantity: 1 },
-  { product_id: 5, quantity: 3 },
+  { product_id: 1, quantity: 2, price: 5 },
+  { product_id: 3, quantity: 1, price: 3 },
+  { product_id: 4, quantity: 1, price: 9 },
+  { product_id: 5, quantity: 3, price: 7 },
```

Comment:
You don't need these lines - the price for each product is queried from the database. (See further comments below.)